spec/unit/processors/os_spec.rb

```diff
@@ -221,6 +221,7 @@ def load(procs)
       before :each do
         proc = stubs 'proc'
         proc.stubs(:Name).returns("Intel(R)    Celeron(R)   processor")
+        proc.stubs(:ole_respond_to?).with(:NumberOfLogicalProcessors).returns(false)
```

Comment:
Additionally you should ensure that `proc.NumberOfLogicalProcessors` is not called.